vars/artifactsPipeline.groovy

```diff
@@ -65,7 +65,8 @@ def call(Map pipelineParams) {
         }
         options {
             timestamps()
-            timeout(pipelineParams.timeout)
+            // Timeout for the whole test, add 30 MINUTES for waiting the builder
+            timeout([time: pipelineParams.timeout.time + 30, unit: 'MINUTES'])
```

Comment:
Due to indentation changes, it is indeed hard to review.
If I understand correctly, the only "real" changes are the +30 here (enough?)
and the step timeout, on line 81 right?

I don't see the description of the "timeout" param above, but it would be nice to write that you add 30 minutes to wait for agent (otherwise user may think it is a problem, that he gave X minutes, and the pipeline is running for more...).

Reply:
> Due to indentation changes, it is indeed hard to review.
> If I understand correctly, the only "real" changes are the +30 here (enough?)
> and the step timeout, on line 81 right?

Yes.

> I don't see the description of the "timeout" param above, but it would be nice to write that you add 30 minutes to wait for agent (otherwise user may think it is a problem, that he gave X minutes, and the pipeline is running for more...).